tests/locators/hw22.spec.js

```diff
@@ -0,0 +1,543 @@
+import {expect, test} from "@playwright/test"
+
+test.describe("Home work locators", () => {
+    test.beforeEach(async () => {
+    })
```

Comment:
add navigation and opening popup to this hook

Reply:
fixed